Cache/Cache.lua

```diff
@@ -23,4 +23,5 @@ function CraftSim.CACHE:ClearAll()
     CraftSim.CACHE.RECIPE_DATA:ClearAll()
     CraftSim.CACHE.ITEM_COUNT:ClearAll()
     CraftSim.CACHE.CRAFT_QUEUE:ClearAll()
+    CraftSim.PRICE_OVERRIDE:ClearAll()

```

Comment:
Consider also changing the behaviour of the clear off of price override to just wipe the main table but just the sub tables (remember the error) :)

Reply:
i made a special cachename check to properly clear the price override cache in the Frames.lua